tools/notary-migration-fix.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+# Copyright Project Harbor Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -ex
+
+harbor_db_image=$(docker images goharbor/harbor-db --format "{{.Repository}}:{{.Tag}}")
+harbor_db_path="/data/database"
+
+launch_db() {
+    docker run -d --name fix-notary-migration -v ${harbor_db_path}:/var/lib/postgresql/data ${harbor_db_image} "postgres"
```

Comment:
In tile env, the docker is '/var/vcap/packages/docker/bin/docker -H unix:///var/vcap/sys/run/docker/dockerd.sock', you need to extract it as variable.

DOCKER_CMD='/var/vcap/packages/docker/bin/docker -H unix:///var/vcap/sys/run/docker/dockerd.sock'
 and use variable in command.
$DOCKER_CMD run  ....

Reply:
I can do it, but how does the script know it's running on tile or vic env? Should I add one more env to identify it? like --env=vic?

Reply:
@stonezdj if you want to reuse this script in tile, you can probably do this, create another `.sh` script and add 
```sh
docker() {
/var/vcap/packages/docker/bin/docker -H unix:///var/vcap/sys/run/docker/dockerd.sock $@
}
source notary-migration-fix.sh
```
such that `docker` is over-written in your script

Reply:
It is OK to me